### PR TITLE
implement 10 tasks week1

### PR DESCRIPTION
**Pull request by Heema5657**
implemented all tasks

### CODE REVIEW

File: databases/week1/data.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
+-- 1 task
+SELECT count(id)
+FROM task
+-- 2 task
+SELECT title, due_date
```
Comment (abedsujan) [inline databases/week1/data.sql]: for task 2: you need to tell how many tasks

File: databases/week1/data.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+-- 4 task
+SELECT *
+FROM TASK
+WHERE STATUS_ID != 3
+-- 5 task
+SELECT *
+FROM TASK
+ORDER BY created ASC
+-- 6 task
+SELECT *
```
Comment (abedsujan) [inline databases/week1/data.sql:21]: the query should list only one task but seems like you are trying to show all the tasks in DESC

File: databases/week1/data.sql [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+-- SELECT TOP 1 *
+-- FROM TASK
+-- ORDER BY created DESC
+-- 7 task
+SELECT title, due_date
+FROM TASK
+WHERE title LIKE '%database%' OR description LIKE '%database%';
+-- 8 task
+SELECT TASK.TITLE, STATUS.NAME
+FROM TASK, STATUS
```
Comment (abedsujan) [inline databases/week1/data.sql]: Try to do by JOIN statement

File: databases/week1/data.sql [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+WHERE (TASK.STATUS_ID = STATUS.ID);
+-- 9 task
+SELECT count(STATUS.NAME), status.name
+FROM TASK, STATUS
+WHERE TASK.STATUS_ID = STATUS.ID
+group by status.name
+-- 10 task
+SELECT count(STATUS.NAME), status.name
```
Comment (abedsujan) [inline databases/week1/data.sql]: use alias for count(STATUS.NAME)

**Review by abedsujan — CHANGES_REQUESTED** [review]

File: databases/week1/data.sql (hunk 2 above)
Comment (Heema5657) [reply databases/week1/data.sql:21]: as I use Limit 1 in the end which only gives 1, it was commented out ! sorry about that. 
I have tried with Top 1 but doesnt work, is there any other suggestion from you

File: databases/week1/data.sql (hunk 3 above)
Comment (Heema5657) [reply databases/week1/data.sql]: Done ! Need some more understanding on Joins.  :)

File: databases/week1/data.sql (hunk 4 above)
Comment (Heema5657) [reply databases/week1/data.sql]: Yeah! It is amazing.

Comment (Heema5657) [reply databases/week1/data.sql]: I have implemented all the changes , can you approve for merge ?

File: databases/week1/data.sql [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+-- 7 task
+SELECT title, due_date
+FROM TASK
+WHERE title LIKE '%database%' OR description LIKE '%database%';
+-- 8 task
+-- SELECT TASK.TITLE, STATUS.NAME
+-- FROM TASK, STATUS
+-- WHERE (TASK.STATUS_ID = STATUS.ID);
+SELECT TASK.TITLE, STATUS.NAME
+FROM (task inner join status )
```
Comment (abedsujan) [inline databases/week1/data.sql:38]: try to improve the join statement
follow some tutorial: https://www.w3schools.com/sql/sql_join.asp 
and come back if you have any confusion understanding

Comment (Heema5657) [reply databases/week1/data.sql:38]: I will practice a more with Joins and try to rewrite it. Thanks 👍

**Review by abedsujan — APPROVED** [review]

**Pull request merged**